packages/block-library/src/query/index.js

```diff
@@ -27,7 +27,7 @@ export const settings = {
 };
 
 export const init = () => {
-	addFilter( 'editor.BlockEdit', 'core/query', queryInspectorControls );
+	addFilter( 'editor.BlockEdit', 'core/query', queryBlockInfo );
```

Comment:
I wonder why a filter is used for this, vs. adding it to the block directly.

Reply:
I have a suspicion it was a means of ensuring the new post link was the first inspector control fill to be rendered. In the earlier PRs addressing this (https://github.com/WordPress/gutenberg/pull/45437/ & https://github.com/WordPress/gutenberg/pull/49566) I removed the filter but I haven't got to it here yet.

Reply:
✅ a40673fac1